parens_2.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import sys
+
+
+def _parens(prefix, left, right):
+    if left == 0 and right == 0: sys.stdout.write(prefix)
```

Comment:
Yes... i see... sys.stdout.write don't append a '\n' at the end, like print does... will change that.